packages/core/src/Resource/Resource.js

```diff
@@ -128,7 +128,23 @@ function Resource(id = null, type, data = null, relationships = null, meta = nul
      * @returns {Object}
      */
     get state() {
-      return currentState(state);
+      const normalState = currentState(state);
```

Comment:
Ik twijfel nog aan de variable naam, normalState is een beetje gek misschien. Kan ook currentState zijn, matched het met wat de functie teruggeeft..

Reply:
ja maar dan ging hij volgens mij miepen over de variabel naam, zal even kijken

Reply:
ja hij zegt "currentState" is already defined in the upper scope

Reply:
Ja natuurlijk, logisch als je erover nadenkt en realiseert dat functions ook gewoon assignments zijn en dus niet overschreven kunnen worden. Prima naam dan bij deze, ik weet iig geen betere